### PR TITLE
[FIX] Regression: re-adding team statistics and new omnichannel statistic

### PR DESCRIPTION
**Pull request by cauefcr**
Sorry data team, the missing stats are on me.

### CODE REVIEW

**Review by KevLehman — APPROVED** [review]

File: app/statistics/server/lib/statistics.ts [hunk 1]
```diff
@@ -449,15 +449,32 @@ export const statistics = {
 			}),
 		);
 
-		statsPms.push(Analytics.resetSeatRequestCount());
+		statsPms.push(
+			new Promise<void>((resolve) => {
+				statistics.totalOTRRooms = Rooms.find({ createdOTR: true }).count();
```
Comment (sampaiodiego) [inline app/statistics/server/lib/statistics.ts:454]: the field `createdOTR` is missing an index and this query should be inside the model.. the method can be named liked `Rooms.findByCreatedOTR()`

File: app/statistics/server/lib/statistics.ts [hunk 2]
```diff
@@ -449,15 +449,32 @@ export const statistics = {
...
+				resolve();
+			}),
+		);
+
+		statsPms.push(
+			new Promise<void>((resolve) => {
```
Comment (sampaiodiego) [inline app/statistics/server/lib/statistics.ts:460]: there is no need for this promise.

File: ee/app/license/server/getStatistics.ts [hunk 3]
```diff
@@ -62,6 +64,13 @@ export async function getStatistics(): Promise<ENTERPRISE_STATISTICS> {
 		}),
 	);
 
+	statsPms.push(
+		new Promise<void>((resolve) => {
+			statistics.businessHourType = settings.get('Livechat_business_hour_type');
```
Comment (sampaiodiego) [inline ee/app/license/server/getStatistics.ts:69]: this doesn't need to be in a promise.

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Review by KevLehman — CHANGES_REQUESTED** [review]
Can you create an index on `message.t`? Looks like we don't have one and that may affect perf (since messages collection is the biggest one)

**Pull request closed without merging**